tests/unit/test_sandbox.py

```diff
@@ -108,8 +104,46 @@ def test_ssh_box_failed_cmd_run_as_devin(temp_dir):
         },
         clear=True
     ):
-        ssh_box = DockerSSHBox()
+        for box in [DockerSSHBox(), DockerExecBox()]:
+            exit_code, output = box.execute('non_existing_command')
+            assert exit_code != 0, 'The exit code should not be 0 for a failed command for ' + box.__class__.__name__
 
-        # test the ssh box with a command that fails
-        exit_code, output = ssh_box.execute('non_existing_command')
-        assert exit_code != 0, 'The exit code should not be 0 for a failed command.'
+def test_single_multiline_command(temp_dir):
+    with patch.dict(
+        config.config,
+        {
+            config.ConfigType.WORKSPACE_BASE: temp_dir,
+            config.ConfigType.RUN_AS_DEVIN: 'true',
+            config.ConfigType.SANDBOX_TYPE: 'ssh',
+        },
+        clear=True
+    ):
+        for box in [DockerSSHBox(), DockerExecBox()]:
+            exit_code, output = box.execute('echo \\\n -e "foo"')
+            assert exit_code == 0, 'The exit code should be 0 for ' + box.__class__.__name__
+            if isinstance(box, DockerExecBox):
+                assert output == 'foo', 'The output should be the same as the input for ' + box.__class__.__name__
+            else:
+                # FIXME: why is there a `>` in the output?
+                assert output == '> foo', 'The output should be the same as the input for ' + box.__class__.__name__
```

Comment:
@xingyaoww this is the only weird bit left. I can't figure out where `>` is coming from. I tried setting PS1, but no luck

Reply:
Best guess is that's actually `PS2` (the continuation prompt). But I can't seem to override it...

Reply:
Same.. can't figure out where it came from..